site/_docs/reference.md

```diff
@@ -2804,6 +2804,7 @@ In the following:
 | b s | DATE_FROM_UNIX_DATE(integer)                 | Returns the DATE that is *integer* days after 1970-01-01
 | p r | DATE_PART(timeUnit, datetime)                | Equivalent to `EXTRACT(timeUnit FROM  datetime)`
 | b | DATE_ADD(date, interval)                       | Returns the DATE value that occurs *interval* after *date*
+| s | DATE_ADD(date, num_days)                       | Returns the DATE that is *num_days* after *date*
```

Comment:
use camelCase for arguments, not snake_case.

Reply:
done